drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/ActionWorkItemPlugin.java

```diff
@@ -173,10 +173,25 @@ public PortableWorkDefinition getWorkItemDefinition() {
     public void forEachWorkItem(final BiConsumer<String, String> biConsumer) {
         getPresenter()
                 .getWorkItemDefinitions()
+                .stream()
+                .filter(this::isWorkItemAvailable)
                 .forEach(workDefinition -> biConsumer.accept(workDefinition.getDisplayName(),
                                                              workDefinition.getName()));
     }
 
+    private boolean isWorkItemAvailable(final PortableWorkDefinition pwd) {
+        if (isNewColumn()) {
```

Comment:
@jomarko Yes I agree. Good find :-)

I only work on JIRAs relating to non-DMN things on Fridays at the moment; so will take another look later this week.